modules/ducktests/tests/ignitetest/services/utils/ignite_spec.py

```diff
@@ -245,12 +245,16 @@ def envs(self):
         """
         :return: environment set.
         """
-        return {
+        environment_dict = {
             'EXCLUDE_TEST_CLASSES': 'true',
             'IGNITE_LOG_DIR': self.service.log_dir,
             'USER_LIBS': ":".join(self.libs()),
             "MAIN_CLASS": self.service.main_java_class
         }
+        if "direct-io" not in self.modules():
```

Comment:
Please  follow here the "Semantic Units" rule from the coding guidelines:
https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-SemanticUnits

It's for java but general requirements are also need to be followed in python code as well.

So, add  empty line before this line.

Reply:
Fixed style.